packages/svelte/tests/runtime-runes/samples/attachment-component-spread/main.svelte

```diff
@@ -0,0 +1,9 @@
+<script>
+	import Child from './Child.svelte';
+
+	let stuff = $state({
+		[Symbol()]: (node) => node.textContent = 'set from component'
+	});
```

Comment:
test should be enhanced to test that when removing the old symbol and adding a new one that the old one is cleaned up properly